test/cases/api.sh

```diff
@@ -407,7 +410,19 @@ function createReqFileAWS() {
           }
       }
     }
-  ]
+  ],
+    "customizations": {
+        "users":[
+            {
+                "name": "user1",
```

Comment:
Could we add 1 of the users to the 'wheel' group and see if it has access to sudo?

Reply:
good idea!

Reply:
So I've tried it:

```
sh toto@ec2-18-193-77-205.eu-central-1.compute.amazonaws.com
[toto@ip-172-31-44-249 ~]$ sudo su
[sudo] password for toto: 
[toto@ip-172-31-44-249 ~]$ groups
toto wheel
```

The issue is that sudo asks for a password, so I could instead lists the groups ?

Reply:
So know the test looks for the presence of wheel for user1 and the absence of wheel for user2.